root/usr/share/nethesis/NethServer/Module/Getmail/Modify.php

```diff
@@ -55,6 +55,7 @@ public function initialize()
             array('Password', Validate::ANYTHING, \Nethgui\Controller\Table\Modify::FIELD),
             array('Server', Validate::HOSTADDRESS, \Nethgui\Controller\Table\Modify::FIELD),
             array('Username', Validate::ANYTHING, \Nethgui\Controller\Table\Modify::FIELD),
+            array('Mailbox', Validate::ANYTHING, \Nethgui\Controller\Table\Modify::FIELD),
```

Comment:
I'd prefer a stronger validation. A similar issue has been addressed here

https://github.com/NethServer/nethserver-mail-common/blob/master/root/usr/share/nethesis/NethServer/Module/Mail/SmtpAccess.php#L37-L56

The ``readAccessBypassList()`` and ``writeAccessBypassList()`` are invoked by the framework when the DB prop is read/written respectively.

The element split procedure also trims surrounding whitespaces.